### PR TITLE
Make akka-protobuf-v3_2.12-2.6.x reproducible again

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -318,7 +318,9 @@ lazy val protobufV3 = akkaModule("akka-protobuf-v3")
     exportJars := true, // in dependent projects, use assembled and shaded jar
     makePomConfiguration := makePomConfiguration.value
         .withConfigurations(Vector(Compile)), // prevent original dependency to be added to pom as runtime dep
-    packagedArtifact in (Compile, packageBin) := Scoped.mkTuple2((artifact in (Compile, packageBin)).value, OsgiKeys.bundle.value),
+    packagedArtifact in (Compile, packageBin) := Scoped.mkTuple2(
+        (artifact in (Compile, packageBin)).value,
+        ReproducibleBuildsPlugin.postProcessJar(OsgiKeys.bundle.value)),
```
Comment (patriknw) [inline build.sbt:323]: osgi changes break stuff again

**Review by patriknw — APPROVED** [review]
LGTM, I guess.
Not the same problem in the other protobuf artifact?

**Comment by raboof** [comment]
> Not the same problem in the other protobuf artifact?

AFAICS no: seems like there we only publish the OSGi bundle and not also the 'normal' artifact

**Review by ennru — APPROVED** [review]
LGTM.

**Pull request merged**